.github/workflows/healthchecks_sqlserver_ci.yml

```diff
@@ -30,14 +30,6 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    services:

```

Comment:
Seeing this change made me realize that we now may be able to do some yaml refactoring and reuse the https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/.github/workflows/reusable_ci_workflow.yml file.

The jobs part for this file could be just:

```yml
jobs:
  build:
    uses: ./.github/workflows/reusable_ci_workflow.yml
    with:
      PROJECT_PATH: ./src/HealthChecks.SqlServer/HealthChecks.SqlServer.csproj
      TEST_PROJECT_PATH: ./test/HealthChecks.SqlServer.Tests/HealthChecks.SqlServer.Tests.csproj
      CODECOV_FLAGS: SqlServer
```

(I took the example from https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/f54852f39646fb5e1615c42278a051fef9181ec3/.github/workflows/healthchecks_cosmosdb_ci.yml#L30-L36)